backup/moodle2/backup_zoom_stepslib.php

```diff
@@ -42,7 +42,7 @@ protected function define_structure() {
             'webinar', 'duration', 'timezone', 'password', 'option_jbh', 'option_start_type', 'option_host_video',
             'option_participants_video', 'option_audio', 'option_mute_upon_entry', 'option_waiting_room',
             'option_authenticated_users', 'option_encryption_type', 'exists_on_zoom', 'alternative_hosts',
-            'recordings_visible_default',
+            'recordings_visible_default', 'show_schedule', 'show_security', 'show_media'
```

Comment:
Because this is an array, it's safe to add a trailing comma (simplifies future diffs)
```suggestion
            'recordings_visible_default', 'show_schedule', 'show_security', 'show_media',
```